c2corg_ui/static/menu-discourse/menu.html

```diff
@@ -0,0 +1,117 @@
+## HEAD
+<link href="//netdna.bootstrapcdn.com/bootstrap/3.0.0/css/bootstrap-glyphicons.css" rel="stylesheet">
+
+## En-tete
+<aside id="sidemenu">
+  <div class="page-menu">
+    <a href="//www.demov6.camptocamp.org">
```

Comment:
> There can be at maximum one <base> element in a document, and it must be inside the <head> element.

So no, there is a dedicated "head" tab in discourse.

There is no problem with my code if i specify that i want to replace links only in `#sidemenu a`.